### PR TITLE
front: stdcm add consist inputs for tonnage and length

### PR DESCRIPTION
**Pull request by SarahBellaha**
Closes [STDCM: Add consist inputs for tonnage and length](https://github.com/osrd-project/osrd-confidential/issues/674)

### CODE REVIEW

File: front/src/applications/stdcmV2/components/StdcmConsist.tsx [hunk 1]
```diff
@@ -140,10 +140,10 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
             onSelectSuggestion={onSelectSuggestion}
           />
         </div>
-        {/* <div className="stdcm-v2-consist__properties d-flex justify-content-between">
-          <Input id="tonnage" label="Tonnage" trailingContent="t" disabled />
-          <Input id="Longueur" label="longueur" trailingContent="m" disabled />
-        </div> */}
+        <div className="stdcm-v2-consist__properties d-flex justify-content-between">
+          <Input id="tonnage" label={t('consist.tonnage')} trailingContent="t" />
+          <Input id="Longueur" label={t('consist.length')} trailingContent="m" />
```
Comment (woshilapin) [inline front/src/applications/stdcmV2/components/StdcmConsist.tsx]:
```suggestion
          <Input id="length" label={t('consist.length')} trailingContent="m" />
```

**Review by woshilapin — APPROVED** [review]
Nice, it's coming too fast.

File: front/src/applications/stdcmV2/components/StdcmConsist.tsx (hunk 1 above)
Comment (SarahBellaha) [reply front/src/applications/stdcmV2/components/StdcmConsist.tsx]: oops, thx ! 👍

**Review by kmer2016 — APPROVED** [review]
LGTM ! Thank you for the PR

**Review by RomainValls — APPROVED** [review]

File: front/src/applications/stdcmV2/components/StdcmConsist.tsx [hunk 2]
```diff
@@ -140,10 +140,10 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
             onSelectSuggestion={onSelectSuggestion}
           />
         </div>
-        {/* <div className="stdcm-v2-consist__properties d-flex justify-content-between">
-          <Input id="tonnage" label="Tonnage" trailingContent="t" disabled />
-          <Input id="Longueur" label="longueur" trailingContent="m" disabled />
-        </div> */}
+        <div className="stdcm-v2-consist__properties d-flex justify-content-between">
+          <Input id="tonnage" label={t('consist.tonnage')} trailingContent="t" />
```
Comment (woshilapin) [inline front/src/applications/stdcmV2/components/StdcmConsist.tsx:144]: Maybe we should name that `mass`?
See https://github.com/OpenRailAssociation/osrd/pull/9106#discussion_r1784022222.

**Pull request merged**